### DISCUSSION TITLE
Fullscreen Drawer

### DISCUSSION THREAD

**Discussion by ivanjeremic** [saadeghi/daisyui#2604] [General]
Is there a Fullscreen Drawer? I find it really annoying that daisy ui components always stop at the basic features they miss always something compared to Chakra for example.

**Comment by saadeghi** [comment]
Using Tailwind CSS `min-h-screen` increases the height to screen.  
It's not a good idea for `drawer` class to do that by default because a lot of times people want to fit other stuff on top or bottom of drawer. For example a navbar or a top banner or a footer.  

A big difference between daisyUI and something like Chakra is that daisyUI provides everything customizable while with something like Chakra you should either use the library's design decisions or fight the built-in styles.  
Making drawer full height is one of those decisions that developer should make based on their layout and content.
